easybuild/easyblocks/f/fftw.py

```diff
@@ -116,6 +116,11 @@ def __init__(self, *args, **kwargs):
                     self.log.info("Enabling use of %s (should be supported based on CPU features)", flag.upper())
                     setattr(self, flag, True)
 
+        # Auto-disable quad-precision on ARM and POWER, as it is unsupported
+        if self.cfg['with_quad_prec'] and arch in [AARCH32, AARCH64, POWER]:
```

Comment:
Well, maybe in the PR to Dylan's repo, but I forget easily. ;-)

I'll make sure to give this a proper test once you've changed this to `cpu_arch`.